### ISSUE TITLE
QC_Report adding Concordance Result

### ISSUE THREAD

**Issue by BenjaminsM** [molgenis/NGS_DNA#132]
In QC report only the header of the concordance output is shown.
So you don't get the actual information you need to check if nothing has been swapped.

Maybe it would be better to show the entire table of the concordance output in the QC Report?

**Comment by RoanKanninga** [comment]
That is correct, I removed the (sequonom) concordance output file since we don't use produce sequonom concordance output, but I forgot to remove the header out of the QC report. 

If the HMF concordance check will be used more often we should put the concordance check results back in the QC report
